tools/UpdateVendors/Program.cs

```diff
@@ -136,6 +144,29 @@ private static void RewriteCsFileWithStandardTransform(string filePath, string o
             Console.WriteLine($"Finished {libraryName} upgrade.");
         }
 
+        private static bool ShouldDropFile(string filePath)
+        {
+            var lowerCasePath = filePath.ToLowerInvariant();
+
+            var drops = new List<string>()
```

Comment:
`lowerCasePath` is no longer used.

```suggestion
        {
            var drops = new List<string>()
```